jupyter-docker/jupyter_notebook_config.py

```diff
@@ -10,4 +10,9 @@
 c.NotebookApp.ip = '*'
 c.NotebookApp.port = 8000
 c.NotebookApp.open_browser = False
-c.NotebookApp.token = ''
\ No newline at end of file
+c.NotebookApp.token = ''
```

Comment:
@ansingh7115 can we rebuild the Jupyter docker image with the changes here?

Reply:
Yes! I will build another image with this change and test it